src/internal/packager/images/pull.go

```diff
@@ -354,24 +356,37 @@ func CleanupInProgressLayers(ctx context.Context, img v1.Image) error {
 
 // SaveSequential saves images sequentially.
 func SaveSequential(ctx context.Context, cl clayout.Path, m map[transform.Image]v1.Image) (map[transform.Image]v1.Image, error) {
+	l := logger.From(ctx)
 	saved := map[transform.Image]v1.Image{}
 	for info, img := range m {
 		annotations := map[string]string{
 			ocispec.AnnotationBaseImageName: info.Reference,
 		}
+		// NOTE(mkcp): It's ok if the metadata is empty / err, they're just for debug logs.
+		wStart := time.Now()
+		size, _ := img.Size() //nolint:errcheck
```

Comment:
IMO we shouldn't skip these errors. Zarf has been checking `img.Size()` on push for the progressbar until recently when we removed it, and I've never seen an error from it. Likely if this breaks, there is something wrong with the image / with the pull

Reply:
My intent was avoiding modifying any of the retry and cleanup behavior but i'm fine returning an error here. Added in next commit